kernel/src/main/scala/cats/kernel/instances/SortedSetInstances.scala

```diff
@@ -28,9 +31,13 @@ class SortedSetOrder[A: Order] extends Order[SortedSet[A]] {
     StaticMethods.iteratorEq(s1.iterator, s2.iterator)
 }
 
-class SortedSetHash[A: Order: Hash] extends Hash[SortedSet[A]] {
+// FIXME use context bound in 3.x
```

Comment:
Just curious, context bound breaks BC here (even with the deprecated constructor)?

Reply:
Yes, because it affects the constructor.